src/guide/frontend/ios.md

```diff
@@ -0,0 +1,72 @@
+---
+title: iOS Client Reference
+---
+
+# iOS Client Reference
+
+The purpose of this SDK is to be able to easily integrate [Passkeys](https://developer.apple.com/passkeys/) into your iOS app with the help of the [Passwordless.dev](https://bitwarden.com/products/passwordless/) management system.
```

Comment:
We probably require an extra section mentioning requirements:
- Permissions & capabilities
- Minimum iOS version

Reply:
Ah good call out. I'll add minimum iOS version and capabilities.

Reply:
Updated.

Reply:
@victor-livefront Related, de-capitalize 'passkeys'